Project Scope Statement.md

```diff
@@ -0,0 +1,12 @@
+﻿| **PROJECT SCOPE STATEMENT** |                                                                                                                                                                                                                                                   |
+|:---------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       **Project Name**      | Threewide                                                                                                                                                                                                                                         |
+|   **Project Deliverables**  | **Detailed Description**                                                                                                                                                                                                                          |
+|        Login/Register       | Threewide will provide a login/register functionality to protect user information and maintain privacy.                                                                                                                                           |
```

Comment:
We will not provide this for either MVP 1. MVP 2 will we have logging in but we will give accounts